azurerm/resource_arm_application_gateway.go

```diff
@@ -2282,6 +2306,42 @@ func flattenApplicationGatewayRedirectConfigurations(input *[]network.Applicatio
 	return results, nil
 }
 
+func expandAutoscaleConfiguration(d *schema.ResourceData) *network.ApplicationGatewayAutoscaleConfiguration {
```

Comment:
Shouldn't this be called expandApplicationGatewayAutoscaleConfiguration for consistency?